lit/examples/nodejs.lit

```diff
@@ -5,43 +5,57 @@
 
 You can run the tests for a Nodejs application.
 
+This also demonstrates seperating the npm install stage from the test running.
 \frame{https://ci.concourse-ci.org/teams/examples/pipelines/nodejs}
 
 \section{
   \title{Pipeline Configuration}{configuration}
 	\codeblock{yaml}{{{
 ---
 resources:
-  - name: nodejs.org-git
+  - name: repo
     type: git
     icon: github
     source:
       uri: https://github.com/nodejs/nodejs.org.git
 
+  - name: node-image
+    type: docker-image
+    source:
+      repository: node
+      tag: 13.10.1-stretch
+      
 jobs:
   - name: test
     public: true
     plan:
-      - get: nodejs.org-git
-        trigger: true
-      - task: run-tests
+      - get: node-image
+      - get: repo
+      - task: install
+        image: node-image
         config:
+          inputs:
+          - name: repo
```

Comment:
Given how often you're using `source` elsewhere, this was probably what you were going for
```suggestion
          - name: repo
            path: source
```